packages/shared/lib/models/Auth.ts

```diff
@@ -129,6 +129,12 @@ export interface OAuth2Credentials extends CredentialsCommon {
 
     refresh_token?: string;
     expires_at?: Date | undefined;
+
+    config_override?: {
+        client_id?: string;
+        client_secret?: string;
+        scopes?: string | string[];
```

Comment:
nitpicking but is it possible to only allow `string[]`?

Reply:
Updated. It is actually just a `string`. It multiple scopes it is comma separated